rutil/test/testCompat.cxx

```diff
@@ -24,6 +24,141 @@ main()
    UInt64 data2 = hton64(data);
    assert(tmp[7] == 1);
 
+
+   long long i1 = 5223372036854775807,i2 = 9223372036854775807;
+   double d1 = 5223372036854775807.9965443,d2 = 9223372036854775807.1265443;
+   long long i11 = -5223372036854775807, i22 = -9223372036854775807;
+   double d11 = -5223372036854775807.99, d22 = -9223372036854775807.12;
+   
+//test for resipMax
+   {
+	   //for postive integer
+	   assert(resipMax(3,2) == 3);
+	   cerr << "max(3,2 ): " << resipMax(3,2)<<endl;
+	   //reverse of above combination
+	   assert(resipMax(2,3) == 3);
+	   cerr << "max(2,3 ): " << resipMax(2,3)<<endl;
+	   //for negative integers
+	   assert(resipMax(-3,-2) == -2);
+	   cerr << "max(-3,-2):" << resipMax(-3,-2)<< endl;
+	   //for large integers
+	   assert(resipMax(i1,i2) == i2);
+	   cerr<< "max(5223372036854775807,9223372036854775807): "<< resipMax(i1,i2)<< endl;
+	   //for large floating point numbers
+	   assert(resipMax(d1,d2) == d2) ;
+	   cerr<< "max(5223372036854775807.9965443,5223372036854775807.1265443 ): "<< resipMax(d1,d2)<< endl;
+	   //for large negative numbers
+	   assert(resipMax(i11,i22) == i11);
+	   cerr<< "max(-5223372036854775807,-9223372036854775807): "<< resipMax(i11,i22)<<endl;
+	   //for large negative floating point numbers
+	   assert(resipMax(d11,d22) == d11);
+	   cerr<< "max(-5223372036854775807.99,-9223372036854775807.12): "<< resipMax(d11,d22)<< endl;
+	   //for same numbers
+	   assert(resipMax(3,3) == 3);
+	   cerr << "max(3,3 ): " << resipMax(3,3)<<endl;
+	   //for char type arguements
+	   assert(resipMax('a','b') == 'b');
+	   cerr<< "max('a','b'): "<<resipMax('a','b')<<endl;
+	   //test for string type arguements
+	   assert(resipMax("hello","jello") == "jello");
```

Comment:
here you are comparing the pointers and not the strings.  Can you either remove these lines of code that compare strings, or represent the strings using a class such as std::string or resip::Data?